checkpoint-multi-nic/azuredeploy.parameters.json

```diff
@@ -12,7 +12,7 @@
             "value": "changeme"
```

Comment:
@chkp-zegman Can you look at putting in placeholder parameters that will allow us to test this template with our CI? https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci